### DISCUSSION TITLE
Preview while moving in a file

### DISCUSSION THREAD

**Discussion by az-z** [airblade/vim-gitgutter#897] [Q&A] [answered]
Hello,
A somewhat silly question, is there a way to show hunks in the popup window while navigating in the file?

It is my current understanding, that to see the changes made I need to navigate to a hunk ( ]c) and fire a preview window ( <leader>hp ).

**Comment by airblade** [comment, marked as answer]
Try this:

```vim
au CursorMoved * if gitgutter#hunk#in_hunk(line(".")) | GitGutterPreviewHunk | else | pclose | endif
```

**Reply by az-z** [reply]
Andy,
 thank you so much. I'm getting the hunks preview at the bottom of the vim, like this :
 
![image](https://github.com/user-attachments/assets/54e95dbb-66ff-4901-ae53-25de440ca2e2)

This will be perfect for small changes, but for the multiline, it will probably be a mess... Is there a way to open hunks in the floating window (I think this is what it is called ... )?

**Comment by airblade** [comment]
Yes, it's documented here:

https://github.com/airblade/vim-gitgutter#to-use-floatingpopup-windows-for-hunk-previews

**Reply by az-z** [reply]
oh! I forgot about that. 
Looks great.
Now I need to make the floating window a lesser (color) disaster :)  
Thank you @airblade !
